c++/src/capnp/compat/http-over-capnp.c++

```diff
@@ -475,7 +475,9 @@ public:
     auto rpcRequest = inner.connectRequest();
     auto downPipe = kj::newOneWayPipe();
     rpcRequest.setHost(host);
-    rpcRequest.setDown(factory.streamFactory.kjToCapnp(kj::mv(downPipe.out)));
+    capnp::ByteStream::Client downByteStream = factory.streamFactory.kjToCapnp(
+        kj::mv(downPipe.out));
+    rpcRequest.setDown(downByteStream);
```

Comment:
Nitpick: This diff looks vestigial? `downByteStream` is not used below. You should `kj::mv(downByteStream)` if you aren't using it later; otherwise this is adding refcounts -- or revert the diff.